discretisedfield/field.py

```diff
@@ -1401,11 +1401,11 @@ def __mul__(self, other):
 
         It can be applied between:
 
-        1. Two fields with equal vector dimentions,
+        1. Two fields with equal vector dimensions ``nvdim``,
```

Comment:
We seem to use a mixture of `value dimensions` and `vector dimensions` when describing `nvdims`.

Reply:
Yes, that is a bit unfortunate. We mean value dimensions but in this particular example those values are vector fields, so the 'v' can be misread. I had noticed but didn't see a good solution.

Perhaps we can get rid of the `vector` here (and potentially replace it by `value`): if both fields are scalar, we can also multiply them?